templates/person/show.json.twig

```diff
@@ -0,0 +1,94 @@
+            {% if person.dob %},"birthDate": "{{person.dob}}" {% endif %}
+            {% if person.dod %}, "deathDate": "{{person.dod}}" {% endif %}
+            {% if person.cityOfBirth %},
+                "birthPlace": {
+                    "@type": "Place",
+                    "name": "{{person.cityOfBirth}}",
+                    "geo": {
+                        "@type": "GeoCoordinates",
+                        "latitude": "{{person.cityOfBirth.latitude}}",
+                        "longitude": "{{person.cityOfBirth.longitude}}"
+                    }
+                }
+             {% endif %}
+            {% if person.cityOfDeath %},
+                "deathPlace": {
+                    "@type": "Place",
+                    "name": "{{person.cityOfDeath}}",
+                    "geo": {
+                        "@type": "GeoCoordinates",
+                        "latitude": "{{person.cityOfDeath.latitude}}",
+                        "longitude": "{{person.cityOfDeath.longitude}}"
+                    }
+                }
+            {% endif %}
+            ,"@id": "{{ url('person_show', {'id':person.id}) }}"
+        }
+        {% for titleRole in person.getTitleRoles("Author") %}
+         {% if titleRole.Role.Name == "Author" %}
+                {% set book = titleRole.title %}
+                {% if book %}
+                ,{
+                    "@type": "Book",
+                    "@id": "{{ url('title_show', {'id':book.id}) }}" ,
+                    "name": "{{book.title}}"
+                    {% if book.genre %} ,"genre": "{{book.genre}}" {% endif %}
+                    {% if book.pubdate %},"datePublished": "{{book.pubdate}}" {% endif %}
+                    {% if book.locationOfPrinting %},
+                        "locationCreated": {
+                                    "@type": "Place",
+                                     "name": "{{book.locationOfPrinting}}",
+                                    "geo": {
+                                        "@type": "GeoCoordinates",
+                                        "latitude": "{{book.locationOfPrinting.latitude}}",
+                                        "longitude": "{{book.locationOfPrinting.longitude}}"
+                                    }
+                                }
+                    {% endif %}
+                     ,"author": [
+                        {% for titleRole in book.getTitleRoles("Author") %}{% set author = titleRole.person %}
+                        {
+                            "type":"person",
+                            "@id": "{{ url('person_show', {'id':author.id}) }}"
+                        }
+                        {% if not loop.last %},
+                        {% endif %}
+
+                        {% endfor %}
+                        ]
+
+                    ,"publisher": [
+                        {% for titleFirmrole in book.getTitleFirmroles("Publisher") %}{% set firm = titleFirmrole.firm %}
```

Comment:
This one also does filtering instead of sorting. At least it's consistently inconsistent.

Reply:
😂😂😂